spec/models/application_record_spec.rb

```diff
@@ -104,6 +104,21 @@
     end
   end
 
+  describe 'interview_count' do
+    before :each do
+      record_1 = create :application_record
+      record_2 = create :application_record
+      create :interview, application_record: record_1
+      @collection = ApplicationRecord.where(id: record_1.id)
```

Comment:
Isn't calling `.interview_count` calling on a ApplicationRecord object? I'm confused as to how that's an AR relation.

If, for whatever reason, the `.where` yielded more than one object (it shouldn't though, because it's on an ID) then this code would break because there would be more than one object in the AR relation.

I still don't see why using `.find` or utilizing `record_1` as an instance variance *wouldn't* be better here.